## P4ADE ## {0} : String method, {1} : Exception e
messagebody381401=Exception dans {0} : {1}
## P4ADE ## {0} : String refName
messagebody381403=Aucune destination d''erreur n''est configur\u00E9e pour le service fonctionnel WS : {0}
messagebody381404=Certaines URL d'adresse de ce service fonctionnel de transport WS sont des URL HTTP ordinaires ; or, la strat\u00E9gie requiert le protocole HTTPS.
## P4ADE ## {0} : String uri
messagebody381405=L''URI {0} de ce service proxy est d\u00E9j\u00E0 utilis\u00E9 par l''application noyau OSB
## P4ADE ## {0} : String uri
messagebody381406=Impossible de cr\u00E9er l''adresse WS entrante avec l''URI absolu {0}
## P4ADE ## {0} : String uri, {1} : String other
messagebody381407=L''URI {0} de ce service proxy est d\u00E9j\u00E0 utilis\u00E9 par un autre service proxy : {1}
## P4ADE ## {0} : String uri
messagebody381408=Les services fonctionnels WS doivent toujours avoir des URI absolus. L''URI indiqu\u00E9 n''est pas absolu : {0}
## P4ADE ## {0} : String uri
messagebody381409=Les mod\u00E8les des services sortants WS ne peuvent \u00EAtre que http et https. Le mod\u00E8le de l''URI indiqu\u00E9 n''est pas valide :{0}
messagebody381410=La strat\u00E9gie du service fonctionnel WS requiert une authentification HTTPS de base, mais aucun compte de service n'a \u00E9t\u00E9 configur\u00E9.
## P4ADE ## {0} : String mappingId
messagebody381411=Il y a eu temporisation pour une entr\u00E9e avec la cl\u00E9 {0} dans la table WsCorrelationTable.
messagebody381412=Une erreur est survenue lors de l'envoi de fa\u00E7on fiable du message au service Web cible fiable.
## P4ADE ## {0} : String uri
messagebody381413=L''URI {0} du service proxy WS se termine par ''/''.
## P4ADE ## {0} : String proxy
messagebody381414=Tentative d''envoi indirect \u00E0 un proxy de transport WS requ\u00E9rant une authentification de base ({0}). Les acheminements de proxy \u00E0 proxy et l''acc\u00E8s \u00E0 la console de test ne sont pas autoris\u00E9s avec ce mod\u00E8le.
## P4ADE ## {0} : String inboundEndpoint
messagebody381415=L''assertion d''identit\u00E9 de jeton X.509 n''est pas activ\u00E9e dans le domaine de s\u00E9curit\u00E9. Un composant d''assertion d''identit\u00E9 avec prise en charge de jeton X.509 activ\u00E9e est requis car le service proxy WS "{0}" comporte une strat\u00E9gie qui requiert une authentification CLIENT-CERT.
## P4ADE ## {0} : String inboundEndpoint
messagebody381416=Le sous-module d''extension de composant de correspondance de nom utilisateur X.509 du composant d''assertion d''identit\u00E9 par d\u00E9faut n''est pas configur\u00E9. Ce composant est requis car le service proxy WS "{0}" comporte une strat\u00E9gie qui requiert une authentification CLIENT-CERT.
## P4ADE ## {0} : String proxy
messagebody381417=Le proxy de transport WS "{0}" est configur\u00E9 avec une strat\u00E9gie requ\u00E9rant des certificats client et est d\u00E9ploy\u00E9 dans un cluster. En g\u00E9n\u00E9ral, on trouve un \u00E9quilibreur de charge entre le client et le cluster. L''\u00E9quilibreur de charge transmet les certificats client au serveur, mais le serveur n''en tient compte que s''il est configur\u00E9 avec l''option "Proxy pour les certificats client activ\u00E9" ; si ce n''est pas le cas, les certificats ne seront pas pris en compte et les demandes \u00E9choueront.
## P4ADE ## {0} : String name
messagebody381418=Echec inattendu lors du traitement d''un message entrant pour l''adresse {0}
messagebody381419=Les URL du service fonctionnel WS doivent commencer par http:// ou https:// et comporter un nom d'h\u00F4te
messagebody381420=La propri\u00E9t\u00E9 Synchrone de la configuration du transport WS a \u00E9t\u00E9 mise \u00E0 niveau avec la valeur M\u00EAme transaction pour la r\u00E9ponse dans la configuration du service.
messagebody381421=La configuration de la file d'attente des erreurs a \u00E9t\u00E9 mise \u00E0 niveau.
## P4ADE ## {0} : String refGlobalName, {1} : Throwable th
messagebody381422=Echec de l''envoi du message d''\u00E9chec de la demande dans la file d''attente des erreurs pour le service {0} avec l''erreur suivante : {1}
